io/src/main/scala/sbt/io/IO.scala

```diff
@@ -604,7 +604,10 @@ object IO {
     if (outputFile.isDirectory)
       sys.error("Specified output file " + outputFile + " is a directory.")
     else {
-      val outputDir = outputFile.getParentFile
+      val outputDir =  Option(outputFile.getParentFile) match {
+        case Some(parentFile) => parentFile
+        case None             => new File(".")
+      }
```

Comment:
In this case, I'd probably avoid wrapping in Option and just do
```scala
 val outputDir =  outputFile.getParentFile match {
   case null             => new File(".")
   case parentFile  => parentFile
}
```
since there is no way for the null to propagate out of the match.

Reply:
Sure will do that.